src/mmw/mmw/settings/layer_settings.py

```diff
@@ -130,27 +130,34 @@
             'code': 'mean_ppt',
             'display': 'Mean Monthly Precipitation',
             'short_display': 'Mean Precip',
-            'css_class_prefix': 'ppt',
             'helptext': 'PRISM monthly mean precipitation.',
             'url': 'https://{s}.tiles.azavea.com/climate/ppt_{month}/{z}/{x}/{y}.png',  # noqa
             'maxNativeZoom': 10,
             'maxZoom': 18,
             'opacity': 0.85,
             'has_opacity_slider': True,
             'time_slider_values': MONTH_CODES,
+            'use_color_ramp': True,
+            'color_ramp_id': 'precipitation-legend',
+            'legend_units_label': 'Precipitation (mm/month)',
+            'legend_unit_breaks': [0, '', 40, '', 80, '', 120, '', 160, '', 500]
         },
         {
             'code': 'mean_temp',
             'display': 'Mean Monthly Temperature',
             'short_display': 'Mean Temp',
-            'css_class_prefix': 'ppt',
             'helptext': 'PRISM monthly mean temperature.',
             'url': 'https://{s}.tiles.azavea.com/climate/tmean_{month}/{z}/{x}/{y}.png',  # noqa
             'maxNativeZoom': 10,
             'maxZoom': 18,
             'opacity': 0.85,
             'has_opacity_slider': True,
             'time_slider_values': MONTH_CODES,
+            'use_color_ramp': True,
+            'color_ramp_id': 'temperature-legend',
+            'legend_units_label': u'Temperature (\xb0 Celsius)',
```

Comment:
Prefer `\xb0C` as the unit

Reply:
Or `&deg;C` if HTML escaping is allowed. Basically, there should be no space between `°` and `C`

Reply:
I switched this to `\xb0C`.